pkg/process/runner/runner.go

```diff
@@ -99,17 +106,17 @@ func NewCollector(sysCfg *sysconfig.Config, hostInfo *checks.HostInfo, enabledCh
 		}
 	}
 
-	return NewCollectorWithChecks(enabledChecks, runRealTime)
+	return NewCollectorWithChecks(enabledChecks, runRealTime, rtNotifierChan)
 }
 
-// NewCollectorWithChecks creates a new Collector
-func NewCollectorWithChecks(checks []checks.Check, runRealTime bool) (*Collector, error) {
+// NewCollectorWithChecks creates a new CheckRunner
+func NewCollectorWithChecks(checks []checks.Check, runRealTime bool, rtNotifierChan <-chan types.RTResponse) (*CheckRunner, error) {
```

Comment:
Looks like we should rename this since it now returns a `CheckRunner`

Reply:
Ah good catch! Renamed